hvplot/__init__.py

```diff
@@ -209,6 +209,54 @@ def _hook_patch_docstrings(backend):
 
 @_wraps(_pn.bind)
 def bind(function, *args, **kwargs):
+    """
+    Returns a *reactive* function that *depends* on widgets, parameters or Python objects.
+    This means that the "reactive" function can be automatically invoked whenever the underlying
+    widget or parameter values change. The widgets can be Panel or IpyWidgets.
+    
+    `hv.bind` can be **used to start your `.interactive` pipeline** by loading data depending
+    on widgets or pameters values.
+    
+    Reference: https://github.com/holoviz/hvplot/issues/825
+
+    :Example:
+
+    >>> import pandas as pd
+    >>> import panel as pn
+
+    >>> import hvplot
+
+    >>> pn.extension()
+
+    >>> experiment = pn.widgets.IntSlider(value=5, start=0, end=10, step=1, name="Experiment")
+    >>> rows = pn.widgets.IntSlider(value=5, start=0, end=10, step=1, name="Rows")
+
+    >>> def get_data(experiment):
+    ...     # Loading data for the given experiment ...
+    ...     return pd.DataFrame({"x": list(range(0,experiment))})
```

Comment:
I understand that you would like the example to be executable, yet I find it not very clear, it's returning a dataframe with one column and whose length depends on `experiment`, that seems unusual. Here's my suggestion, that requires numpy to be imported.

```suggestion
    >>> def algorithm(alpha):
    ...         # Running an algorithm that uses alpha...
    ...         return pd.DataFrame({"output": np.array([0, 3, 5, 2]) ** alpha})
```

This points to an important point: the docstring should tell why one would need to use `hvplot.bind`, i.e. as the data you want to explorer in an interactive pipeline can be the result of some other preprocessing pipeline (running a simulation, getting data from a database, etc.).